ingestion/src/metadata/ingestion/source/dashboard/metabase/models.py

```diff
@@ -0,0 +1,17 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Metabase Models
+"""
+from pydantic import BaseModel
+
+class MetabaseModel(BaseModel):
+    pass #TODO: find out MetabaseModel data structure and fill in using pydantic types
```

Comment:
Could you guide me on what would comprise of a MetabaseModel, I am still trying to figure this out ? 
@pmbrull  @ulixius9

Reply:
This is to model whatever Metabase is returning us. You can get metabase up and running in docker and play a bit with the responses https://www.metabase.com/. We just want to make sure we are not handling dicts, but rather properly typed elements